e2e_tests/tests/experiment/experiment.py

```diff
@@ -914,3 +914,20 @@ def s3_checkpoint_config_no_creds() -> Dict[str, str]:
 
 def root_user_home_bind_mount() -> Dict[str, str]:
     return {"host_path": "/tmp", "container_path": "/root"}
+
+
+def has_at_least_one_checkpoint(exp_id: int) -> bool:
```

Comment:
Can you just update the code to check if the experiment has any checkpoints?

It is weird that the behavior and the name are not aligned. And I think a function that just checks the most recent trial for a checkpoint is probably not generally useful, anyway.

Reply:
Updated code as per the suggestion.